homework2/tests/task4_test.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any, Tuple
+
+import pytest
+
+from homework2.hw4 import cache
+
+
+@pytest.mark.parametrize("some", [(100, 200), (1, 1), (0, 1)])
+def test_get_most_common_non_ascii_char(some: Tuple[Any, Any]):
```

Comment:
This test didn't check if the value were cached or not.

Reply:
This is the task🤔

Reply:
Done